homeassistant/components/lutron_caseta/switch.py

```diff
@@ -100,6 +100,10 @@ async def async_added_to_hass(self) -> None:
         await super().async_added_to_hass()
         self._smartbridge.add_smart_away_subscriber(self._handle_bridge_update)
 
+    def _handle_bridge_update(self, smart_away_state: str | None = None) -> None:
+        """Handle updated smart away state from the bridge."""
+        self.async_write_ha_state()
```

Comment:
This overrides the one in entity.py which a different signature. 

This creates a confusing behavior because `async_added_to_hass` will still set up a callback to `_handle_bridge_update`, and while it will work because it defaults to `None` it would be cleaner to name this something else and change it to use the new function above